ibis/backends/sqlite/tests/conftest.py

```diff
@@ -2,7 +2,7 @@
 
 import pytest
 
-import ibis
+from ... import sqlite
```

Comment:
You can revert this. Sorry I was not clear, the imports that would be nice to have relative are the imports inside the backend that import things inside the backend.